src/misc/check_keys.jl

```diff
@@ -0,0 +1,29 @@
+"""
+Throw an ArgumentError if model.field has any keys in xs::Union{Reaction, Metabolite, Gene}.
+"""
+function throw_argerror_if_key_found(model, field, xs)
```

Comment:
name: `arg_keys_exists`

use:
`arg_keys_exists(model.field, ks)`

Reply:
need `field` to generate the nice error message.